tests/conftest.py

```diff
@@ -146,7 +146,22 @@ def img_tbl_exprs(indexed_img_tbl: catalog.Table) -> list[exprs.Expr]:
         t.img.rotate(90).resize([224, 224]),
         t.img.fileurl,
         t.img.localpath,
+    ]
+
+@pytest.fixture(scope='function')
+def sim_exprs_same_idx(indexed_img_tbl: catalog.Table) -> list[exprs.Expr]:
+    t = indexed_img_tbl
+    return [
         t.img.similarity('red truck'),
+        t.img.similarity('red truck', idx='img_idx0')
+    ]
+
+@pytest.fixture(scope='function')
+def sim_exprs_diff_idx(multi_idx_img_tbl: catalog.Table) -> list[exprs.Expr]:
+    t = multi_idx_img_tbl
+    return [
+        t.img.similarity('red truck', idx='img_idx1'),
```

Comment:
why not multi_idx_img_tbl_exprs?

Reply:
because it only had similarity exprs. I have changed the name, it fits better. thanks!
<!-- Reviewable comment -OETQn11E0FlAfNZgVlT:b60imrc -->
<!-- Sent from Reviewable.io -->